_posts/blog/2018-10-11-review-of-periodicos-e-literatura.md

```diff
@@ -0,0 +1,27 @@
+
+In order to understand the collection, it's important to understand the organizing principles of the larger news archive from which the materials have been selected. This is in part because the coordinators of the project are part of the staff of the NDLB. The Hemeroteca's role is also important because the NDLB was conceived as an extension of the Foundation; differing only in that it is also responsible for uniting and completing collections housed in different institutions and facilitating digital collection development for partners. The constitutive goals of [*preserving national memory and activating its other collections*](http://bndigital.bn.gov.br/sobre-a-bndigital/missao/) is explicitly privileged as well in this project. However, the introduction of the digital is framed ambivalently and this results in issues discussed later. 
+
+The collection appears to consciously serve two communities: library staff and humanities researchers. The majority of the publications coming from the collection have been [*technical and research papers*]() in LIS journals. The Foundation is the biggest cultural heritage repository in the country and the NDLB shares this position as its digital parallel. All this to say that, the impact of the collection is significant considering its status as an example digital project development in Brazil and the fact that it has the potential to link itself with almost 2 million documents across thousands of other collections housed in the archives. There is also a productive tension between these communities and disciplines that can be summarized as an issue of "essences", as Ann Stoler proposes the term in (*Along the Archival Grain: Epistemic Anxieties and Colonial Commonsense* 4). 
```

Comment:
Missing URL.

Reply:
The final sentence seems to be about a different topic than the rest of the paragraph ("there is a productive tension"). Aside from that part, this paragraph seems to be very clearly about the impact of the collection. Perhaps the discussion of 'essence' could be moved to the next paragraph.